test/compile/layout.test.ts

```diff
@@ -44,7 +44,7 @@ describe('compile/layout', () => {
       });
 
       const sizeExpr = unitSizeExpr(model, X);
-      assert.equal(sizeExpr, 'max(datum["distinct_a"] + 2*0.5, 0) * 21');
+      assert.equal(sizeExpr, 'max(datum["distinct_a"] - 1 + 2*0.5, 0) * 21');
```

Comment:
I guess I would like to see a comment that explains that 0.5 is the passingOuter.